src/20_Components/amp-analytics.html

```diff
@@ -0,0 +1,222 @@
+        "on": "visible",
+        "request": "event",
+        "vars": {
+          "eventId": "pageview"
+        }
+      }
+    }
+  }
+  </script>
+  </amp-analytics>
+
+  <!--
+    This is a more sophisticated view tracking. This sample fires an analytics request when a specific element becomes visible. The element is specified via `id` and must be an AMP element (amp-img, amp-iframe, amp-ad,...). 
+  -->
+  <amp-analytics>
+  <script type="application/json">
+  {
+    "requests": {
+      "event": "https://amp-publisher-samples-staging.herokuapp.com/amp-analytics/ping?user={{.}}&account=ampbyexample&event=${eventId}"
+    },
+    "triggers": {
+      "trackPageview": {
+        "on": "visible",
+        "request": "event",
+        "visibilitySpec": {
+          "selector": "#cat-image-id",
+          "visiblePercentageMin": 20,
+          "totalTimeMin": 500,
+          "continuousTimeMin": 200
+        },
+        "vars": {
+          "eventId": "catview"
+        }
+      }
+    }
+  }
+  </script>
+  </amp-analytics>
+
+  <!--
+    This image will trigger an analytics request, if at least 20% of it are visible for 500ms with a minimum time of 200ms continuous visibility.
+  -->
+  <amp-img id="cat-image-id" src="/img/cat-looking-up-300x200.jpg" width="300" height="200" attribution="visualhunt" alt="a cat" layout="responsive"></amp-img>
+
+  <!-- #### Anchor Clicks -->
+  <!--
+    User interactions can be tracked as well. This configuration will track clicks on any anchor links in the page.
+  -->
+  <amp-analytics>
+  <script type="application/json">
+  {
+    "requests": {
+      "event": "https://amp-publisher-samples-staging.herokuapp.com/amp-analytics/ping?user={{.}}&account=ampbyexample&event=${eventId}"
+    },
+    "triggers": {
+      "trackAnchorClicks": {
+        "on": "click",
+        "selector": "a",
+        "request": "event",
+        "vars": {
+          "eventId": "clickOnAnyAnchor"
+        }
+      }
+    }
+  }
+  </script>
+  </amp-analytics>
+
+  <!-- Try it by clicking one of the links. -->
+  <ul>
+    <li><a>a link</a></li>
+    <li><a>another link</a></li>
+  </ul>
+
+  <!--
+  It is also possible to fire click events only for specific links. Select specific elements to tracking by specifying a `selector` id. This sample only tracks clicks on anchors with the id `anchor-id`.
+  -->
+  <amp-analytics>
+  <script type="application/json">
+  {
+    "requests": {
+      "event": "https://amp-publisher-samples-staging.herokuapp.com/amp-analytics/ping?user={{.}}&account=ampbyexample&event=${eventId}"
+    },
+    "triggers": {
+      "trackAnchorClicks": {
+        "on": "click",
+        "selector": "#anchor-id",
+        "request": "event",
+        "vars": {
+          "eventId": "clickOnSpecialAnchor"
+        }
+      }
+    }
+  }
+  </script>
+  </amp-analytics>
+  <!--  Only one of these links will trigger the `clickOnSpecialAnchor` event. -->
+  <a id="anchor-id">a special link</a>.
```

Comment:
What about using a list like line 141 ( <!-- Try it by clicking one of the links.....)? Or maybe remove the list from the above, just to be consistent.